packages/@aws-cdk/aws-ecs/lib/ec2/ec2-task-definition.ts

```diff
@@ -23,6 +23,20 @@ export interface Ec2TaskDefinitionProps extends CommonTaskDefinitionProps {
    * @default - No placement constraints.
    */
   readonly placementConstraints?: PlacementConstraint[];
+
+  /**
+   * The IPC resource namespace to use for the containers in the task.
+   *
+   * @default - IpcMode used by the task is not specified
+   */
+  readonly ipcMode?: IpcMode;
+
+  /**
+   * The process namespace to use for the containers in the task.
+   *
+   * @default - IpcMode used by the task is not specified
+   */
+  readonly pidMode?: PidMode;
```

Comment:
Can you add the comments here about not being supported by fargate or windows containers?